src/features/surveys/l10n/messageIds.ts

```diff
@@ -9,6 +9,15 @@ export default makeMessages('feat.surveys', {
     textButton: m('Text'),
     title: m('Choose a block type to add more content to your survey'),
   },
+  alert: {
+    editable: m<{ orgTitle: string }>(
+      'This survey is owned by {orgTitle} and is editable only from its original location.'
+    ),
+    goOriginal: m('Go to original'),
+    notEditable: m<{ orgTitle: string }>(
+      'This survey is owned by {orgTitle} and it is not editable.'
```

Comment:
Maybe a more direct text would be:

```suggestion
      'This survey is owned by {orgTitle} and you can not edit it.'
```